plotly/api/v2/dashboards.py

```diff
@@ -0,0 +1,47 @@
+"""
+Interface to Plotly's /v2/dashboards endpoints.
+
+Partially complete at the moment. Only being used by
+plotly.plotly.dashboard_ops.
+"""
+from __future__ import absolute_import
+
+from plotly.api.v2.utils import build_url, request
+
+RESOURCE = 'dashboards'
+
+
+def create(body):
+    """Create a dashboard."""
+    url = build_url(RESOURCE)
+    return request('post', url, json=body)
+
+
+def list():
+    """Returns the list of all users' dashboards."""
+    url = build_url(RESOURCE)
+    return request('get', url)
+
+
+def retrieve(fid, share_key=None):
+    """Retrieve a dashboard from Plotly."""
+    url = build_url(RESOURCE, id=fid)
+    return request('get', url)
+
+
+def update(fid):
+    """Completely update the writable."""
+    url = build_url(RESOURCE, id=fid)
+    return request('put', url)
```

Comment:
How so? I briefly looked at https://api.plot.ly/v2/dashboards#update but yeah, I was going to attempt fixing the overwrite-filename limitation on upload, but just threw update in there to start